tests/test_main.py

```diff
@@ -568,8 +584,31 @@ def test_determine_species_based_on_sa():
                      set_paths=True,
                      )
     t3.rmg_species, t3.rmg_reactions = t3.load_species_and_reactions_from_chemkin_file()
+    sa_observables = ['H2', 'OH']
+    simulate_adapter = simulate_factory(simulate_method=t3.t3['sensitivity']['adapter'],
+                                        t3=t3.t3,
+                                        rmg=t3.rmg,
+                                        paths=t3.paths,
+                                        logger=t3.logger,
+                                        atol=t3.rmg['model']['atol'],
+                                        rtol=t3.rmg['model']['rtol'],
+                                        observable_list=sa_observables,
+                                        sa_atol=t3.t3['sensitivity']['atol'],
+                                        sa_rtol=t3.t3['sensitivity']['atol'],
```

Comment:
Typo: this should say `rtol`. I will push this fix and address the unused import from LGTM with any additional feedback so that I only push once